backend-api/src/functions/common/session/sessionsDemo.test.ts

```diff
@@ -0,0 +1,70 @@
+import { SessionState } from "./Session";
+import { DynamoDbAdapter } from "../../adapters/dynamoDbAdapter";
+import { randomUUID } from "crypto";
+import "dotenv/config";
+import { BiometricSessionFinished } from "./updateOperations/BiometricSessionFinished";
+import { BiometricTokenIssued } from "./updateOperations/BiometricTokenIssued";
+
+describe("Sessions", () => {
```

Comment:
Very clear console.logs produced here, thanks! Wish I had read line 11 a little harder before running the test a few times 😆